front-end/src/app/login/security-notice/security-notice.component.html

```diff
@@ -34,13 +34,13 @@ <h2>Disclaimer</h2>
     <p>While using this system please do not:</p>
     <ul>
       <p>
-        <li>prevent other users from accessing FECFile Online;</li>
-        <li>overload FECFile Online;</li>
+        <li>prevent other users from accessing FECfile+ Online;</li>
+        <li>overload FECfile+ Online;</li>
         <li>share committee or user accounts;</li>
         <li>
-          attempt to gain unauthorized access to FECFile Online or related systems or networks or to defeat, avoid,
+          attempt to gain unauthorized access to FECfile+ Online or related systems or networks or to defeat, avoid,
```

Comment:
I think we want to remove "Online" from anywhere we are using "Fecfile+" now (this may need to be confirmed with the analysts though).  It looks like we are using "Fecfile+ Online" in a number of places.